config.test.yaml

```diff
@@ -38,6 +38,11 @@ default_project: &default_project
           host: http://appservice.wmflabs.org
         events: {}
         purged_cache_control: test_purged_cache_control
+        pdf:
+          # Cache PDF for 5 minutes since it's not purged
+          cache_control: s-maxage=600, max-age=600
+          uri: https://pdf-electron.wmflabs.org/pdf
```

Comment:
We have kind of gotten into the habit of putting only the `protocol://host:port` part in the parameters, so let's keep the consistency with the other config options. Additionally, if we ever decide to use the other formats supported by the service (png, jpg), we won't need to change the options in the config.